modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Secure Network Observability with a network policy using the web console
+You may need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
+
+.Procedure
+. Navigate to *Networking* -> *NetworkPolicies*.
+. Select the `netobserv` project from the *Project* dropdown menu.
+. Name the policy. For this example, the policy name is `allow-ingress`.
+. Click the *Add ingress rule* button twice to create two ingress rules, and make the following specifications in the form:
```

Comment:
I don't work with our GUIs a lot, so this might be ignorance, but why "Click the **Add ingress rule** button twice" rather than "Click **Add ingress rule** twice"?

Reply:
"Make the following specifications" or "Specify"?

Reply:
This is a little challenging but basically you have to click "Add ingress rule" two times to populate two forms that need to be filled in.